### ISSUE TITLE
As a screen reader user, I would like my menus to have a heading I can use to navigate

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/express#1642]
## Context

We got some feedback from screen reader users that the main menu of sites does not have a heading, so it could make it difficult to navigate to when using assistive technology.
